### PR TITLE
Passes existing fein when updating broker profile.

### PR DESCRIPTION
**Pull request by ymhari**
RM-96881

### CODE REVIEW

File: components/benefit_sponsors/app/views/benefit_sponsors/shared/organizations/_entity_name_and_type_fields.html.erb [hunk 1]
```diff
@@ -1,6 +1,9 @@
-<div class="col-md-8 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
-  <%= f.text_field :legal_name, class: "floatlabel form-control interaction-field-control-organization-legal-name typeahead_turned_off", required: true, placeholder: 'LEGAL NAME' %>
-</div>
-<div class="col-md-4 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
-  <%= f.text_field :dba, class: "floatlabel form-control interaction-field-control-organization-dba", placeholder: 'DBA' %>
+<div>
+  <div class="col-md-8 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
+    <%= f.text_field :legal_name, class: "floatlabel form-control interaction-field-control-organization-legal-name typeahead_turned_off", required: true, placeholder: 'LEGAL NAME' %>
```
Comment (markgoho) [inline components/benefit_sponsors/app/views/benefit_sponsors/shared/organizations/_entity_name_and_type_fields.html.erb:3]: placeholder needs translation

File: components/benefit_sponsors/app/views/benefit_sponsors/shared/organizations/_entity_name_and_type_fields.html.erb [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+  </div>
+  <div class="col-md-4 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
+    <%= f.text_field :dba, class: "floatlabel form-control interaction-field-control-organization-dba", placeholder: 'DBA' %>
```
Comment (markgoho) [inline components/benefit_sponsors/app/views/benefit_sponsors/shared/organizations/_entity_name_and_type_fields.html.erb:6]: placeholder needs translation

**Review by markgoho — DISMISSED** [review]
missing translations

**Review by vkghub — APPROVED** [review]
fein is now visible when redirecting back to broker home page.

**Pull request merged**